tasks/run_openstack_service_setup_playbook.yml

```diff
@@ -0,0 +1,21 @@
+---
+# tasks file for molecule-validate-nova-deploy
+
+- name: Find the proper inventory file
+  shell: find /opt/openstack-ansible -name dynamic_inventory.py -print
+  register: find_inventory_file
+  ignore_errors: True
+
+- name: Set proper inventory file
+  set_fact:
+    inventory_file: "{{ find_inventory_file.stdout }}"
+
+- name: Create networks
```

Comment:
This is probably a misleading name, since the `openstack-service-setup.yml` playbook sets up much more than just networks.